### ISSUE TITLE
Fully replicated your code in Chapter 4, doesn't work

### ISSUE THREAD

**Issue by bifork** [habuma/spring-in-action-5-samples#31]
Switching to another course now. Regret buying this book. Example not practical enough. Plus so many errors and unmentioned errors. 
Not professional.

**Comment by habuma** [comment]
Stating that it doesn't work isn't productive at all. Please, tell me what doesn't work and let's work together to find a solution.

**Comment by habuma** [comment]
To be sure, I just now ran through chapter 4's code and although I am aware of a small issue with regard to form validation that isn't handled as elegantly as it should be (on my TODO list to fix it), I was able to register, sign in, create a taco, and submit an order. 

And I'd be curious to hear what isn't practical about the example? Sure, it's quite simple at this stage of the book, but it evolves as the book progresses. What would make it more practical for you?
